From: Jordan Petridis <jpetridis@gnome.org>
Date: Fri, 24 Mar 2023 09:48:47 +0000
Subject: appFavorite: Rename simple-scan.desktop

simple-scan app-id will soon be renamed to org.gnome.SimpleScan
and thus we need to add the .desktop file to `RENAMED_DESKTOP_IDS`

https://gitlab.gnome.org/GNOME/simple-scan/-/merge_requests/11

Part-of: <https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/2723>

Origin: https://gitlab.gnome.org/GNOME/gnome-shell/-/commit/cbecc1d
---
 js/ui/appFavorites.js | 1 +
 1 file changed, 1 insertion(+)

diff --git a/js/ui/appFavorites.js b/js/ui/appFavorites.js
index d8a3018..81a5642 100644
--- a/js/ui/appFavorites.js
+++ b/js/ui/appFavorites.js
@@ -59,6 +59,7 @@ const RENAMED_DESKTOP_IDS = {
     'polari.desktop': 'org.gnome.Polari.desktop',
     'seahorse.desktop': 'org.gnome.seahorse.Application.desktop',
     'shotwell.desktop': 'org.gnome.Shotwell.desktop',
+    'simple-scan.desktop': 'org.gnome.SimpleScan',
     'tali.desktop': 'org.gnome.Tali.desktop',
     'totem.desktop': 'org.gnome.Totem.desktop',
     'evince.desktop': 'org.gnome.Evince.desktop',
